lib/miq_expression/field.rb

```diff
@@ -82,8 +82,20 @@ def contains(other)
     )
   end
 
+  def column_type
+    if custom_attribute_column?
+      CustomAttribute.find_by(:name => custom_attribute_column_name).field_type
```

Comment:
Just thought of something else, maybe this should be scope to the a particular`resource_type` based on the class portion of the column name to prevent finding a custom attribute of the same name owned by a different object class.